docs/vm-installation.md

```diff
@@ -0,0 +1,144 @@
+   antrea-agent.conf file to set `clientConnection`,`antreaClientConnection` and `externalNodeNamespace` with the
+   correct values. Please refer to this [base configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
+   to check all the configuration parameters for VMs.
+
+```bash
+AGENT_NAMESPACE="vm-ns"
+KUBECONFIG_PATH="/var/run/antrea"
+# Update clientConnection and antreaClientConnection
+sed -i "s|kubeconfig: |kubeconfig: $KUBECONFIG_PATH/|g" antrea-agent.conf
+# Update externalNodeNamespace
+sed -i "s|#externalNodeNamespace: default|externalNodeNamespace: $AGENT_NAMESPACE|g" antrea-agent.conf
+```
+
+3. Create `antrea-agent` service. Below is a sample snippet to start `antrea-agent` as a service on Ubuntu 18.04 or
+   later:
+
+```bash
+AGENT_BIN_PATH="/usr/sbin"
+AGENT_CONF_PATH="/var/run/antrea"
+AGENT_LOG_PATH="/var/log/antrea"
+mkdir -p /var/log/antrea
+cat << EOF > /etc/systemd/system/antrea-agent.service
+Description="antrea-agent as a systemd service"
+After=network.target
+[Service]
+ExecStart=$AGENT_BIN_PATH/antrea-agent \
+--config=$AGENT_CONF_PATH/antrea-agent.conf \
+--logtostderr=false \
+--log_file=$AGENT_LOG_PATH/antrea-agent.log
+Restart=on-failure
+[Install]
+WantedBy=multi-user.target
+EOF
+
+sudo systemctl daemon-reload
+sudo systemctl enable antrea-agent
+sudo systemctl start antrea-agent
+```
+
+## Installation on Windows VM
+
+### Prerequisites
+
+1. Enable the Windows Hyper-V optional feature on Windows VM.
+
+```powershell
+Install-WindowsFeature Hyper-V-Powershell
+Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All -NoRestart
+```
+
+2. OVS needs to be installed on the VM. For more information about OVS installation please refer to the [Antrea Windows documentation](windows.md#1-optional-install-ovs-provided-by-antrea-or-your-own).
+3. Download [nssm](https://nssm.cc/download) which will be used to create the Windows service for `antrea-agent`.
+
+Note: Only Windows Server 2019 is supported in the first release at the moment.
+
+### Installation
+
+1. Build `antrea-agent` binary in the root of the antrea code tree and copy the binary to Windows VM.
+
+```bash
+#! /bin/bash
+make docker-windows-bin
+```
+
+2. Copy and update the `antrea-agent` configuration file which specifies agent configuration parameters. Edit
+   antrea-agent.conf file to set `clientConnection`,`antreaClientConnection` and `externalNodeNamespace` with the
+   correct values. Please refer to this [base configuration file](../build/yamls/externalnode/conf/antrea-agent.conf)
+   to check all the configuration parameters for VMs.
+
+3. Create `antrea-agent` service using nssm. Below is a sample snippet to start `antrea-agent` as a service:
+
+```powershell
+$WIN_AGENT_BIN_PATH="C:\antrea-agent"
+$WIN_AGENT_CONF_PATH="C:\antrea-agent\conf"
+$WIN_AGENT_LOG_PATH="C:\ProgramData\Antrea\"
+mkdir -p $AGENT_LOG_PATH
```

Comment:
Remove "-p". It is not supported on Windows.

Reply:
`$WIN_AGENT_BIN_PATH="C:\antrea-agent"` is not the right way to define a variable on Windows? Could you check?

Reply:
This is how we define a variable in powershell. 
the mkdir -p works on powershell, but if the directory exist it will throw an error. 
I will use the native powershell command to create a direcytory.